### PR TITLE
Pr 1548

### PR DESCRIPTION
**Pull request by marco-leo**

### CODE REVIEW

File: commerce-payment-service/src/main/java/com/liferay/commerce/payment/service/impl/CommercePaymentMethodGroupRelLocalServiceImpl.java [hunk 1]
```diff
@@ -279,6 +308,77 @@ public CommercePaymentMethodGroupRel setActive(
...
+				long groupId = commercePaymentMethodGroupRel.getGroupId();
+
+				attributes.put("commerceChannelGroupId", groupId);
+
+				searchContext.setAttributes(attributes);
+
+				searchContext.setCompanyId(
+					commercePaymentMethodGroupRel.getCompanyId());
+
+				CPDataSourceResult cpDataSourceResult =
```
Comment (marco-leo) [inline commerce-payment-service/src/main/java/com/liferay/commerce/payment/service/impl/CommercePaymentMethodGroupRelLocalServiceImpl.java:358]: @alecsloan why we need this?

**Comment by ethanbustad** [comment]
ci:test

File: commerce-payment-service/src/main/java/com/liferay/commerce/payment/service/impl/CommercePaymentMethodGroupRelLocalServiceImpl.java (hunk 1 above)
Comment (alecsloan) [reply commerce-payment-service/src/main/java/com/liferay/commerce/payment/service/impl/CommercePaymentMethodGroupRelLocalServiceImpl.java:358]: If the admin is disabling the last recurring payment method we disable subscriptions on all products where subscription is enabled.

**Comment by alecsloan** [comment]
@marco-leo see #1760

**Pull request closed without merging**